lib/puppet/pops/validation/validator_factory_4_0.rb

```diff
@@ -27,6 +27,7 @@ def severity_producer
 
     p[Issues::FUTURE_RESERVED_WORD]          = :deprecation
 
+    p[Issues::DUPLICATE_KEY]                 = :warning
```

Comment:
This should be rebased on PUP-5889 and then the level should be set to ```Puppet[:string]``` (the value is a compatible symbol). This allows the user to control strictness and if it is an error, warning or ignore.